keyboards/handwired/videowriter/rules.mk

```diff
@@ -0,0 +1,26 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+
```

Comment:
```suggestion
#   Teensy       halfkay
#   Pro Micro    caterina
#   Atmel DFU    atmel-dfu
#   LUFA DFU     lufa-dfu
#   QMK DFU      qmk-dfu
#   ATmega32A    bootloadHID
#   ATmega328P   USBasp
```

Reply:
I usually reflash all my Pro Micros to DFU bootloader, but you're right. People will be using stock Caterina bootloader. I'll make corresponding changes.